docs/PORTFOLIO-SPEC.md

```
@@ -72,6 +72,44 @@ Complex problems, Structured systems.
   - MOS Web Portal 프로젝트 CLI 없이 0부터 세팅
 - 기술 스택 섹션 (카테고리별 정리)
 
+**Tech Stack 상세**
+
+### 🌐 Frontend (Core & Libraries)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)